src/web/rustdoc.rs

```diff
@@ -593,13 +652,14 @@ mod test {
             // check it searches for removed pages
             let redirect =
                 latest_version_redirect("/dummy/0.1.0/dummy/struct.will-be-deleted.html", &web)?;
-            assert_eq!(redirect, "/dummy/0.2.0/dummy?search=will-be-deleted");
+            // This must be a double redirect to deal with crates that failed to build in the
+            // latest version
+            assert_eq!(redirect, "/dummy/0.2.0/?search=will-be-deleted");
```

Comment:
Why did this redirect change?

Reply:
The extra `target_name` here is useless information that isn't needed for the redirect. https://github.com/rust-lang/docs.rs/pull/715/files#diff-db933256db133927163e75c554ffdfb6R197 will strip it, but it can just be not sent anyway.